checklist.tex

```diff
@@ -0,0 +1,36 @@
+\newlist{todolist}{itemize}{2}
+\setlist[todolist]{label=$\square$}
+
+\section{Checklist for iDAC}\label{sec:checklist}
+
+There are a spectrum of possibilities for a DAC from Just hosting Object lite to full copy.
+Here an attempt is made to have a set of check lists  that can be used to look at an iDAC.
+\secref{sec:fullDAC} covers a full up iDAC while  \secref{sec:liteDAC} is the minimum.
+There will undoubtedly be proposals in between.
+
+\subsection{Any DAC} \label{sec:anyDAC}
+\begin{todolist}
+\item Authentication/Authorization system  inline with Rubin Observatory Access
+\item Agreement to make broadly accessible to all Data Rights holders
+\end{todolist}
+
+\subsection{Lite iDAC} \label{sec:liteDAC}
+Items in \secref{sec:anyDAC} as well as :
+\begin{todolist}
+\item Database system capable of handling $4^{10}$ rows.
+\item IVOA TAP interface, MyDB and Table Upload, CAOM support.
+\item About 500TB of disk for catalogs + MyDBs.
+\item Professional support staff (min 0.25 FTE)
+\item Sufficient connectivity to support users
+\end{todolist}
+
+\subsection{Full DAC} \label{sec:fullDAC}
+Items in \secref{sec:anyDAC} as well as :
+\begin{todolist}
+\item Staff (about 5 FTE) to handle major hardware installation
+\item Agreement to stand up standard Science Platform (Puppet/Kubernetes etc.)
+\item Database system capable of handling all catalogs (or Qserv) with IVOA  interfaces.
+\item Understanding of sizing model in \citeds{DMTN-135} - sizing model and a cost model for the iDAC.
+\item Sufficient funding for 10 years of operations  based on cost model and FTE (probably $>\$6M/year$).
```

Comment:
Better to stipulate sufficient funding to cover the LSST operations period, FY22-FY34, I think? This is a hard ask: we may need to soften the wording a little. No-one gets funded for 10 years up front.

Reply:
```suggestion
\item "Commitment to fund the IDAC through the LSST operations period, FY22-FY34 (probably  $>\$6M/year$ based on hardware cost model and labor plan).".
```